Sources/Moya/Task+Internal.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+import Moya
+
+extension Task {
+    typealias TaskParameters = (Encodable, ParameterEncoder)
```

Comment:
nit: Why not `Parameters` instead of `TaskParamters`?

Reply:
I think it was initially because Alamofire already has a `Parameters` type and I didn't want to mix them. Renaming it could indeed be a god idea.

Reply:
Right, but this `Parameters` typealias  **will not** be globally defined.